tools/mkwamrglue.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python3
+############################################################################
+# apps/tools/mkwamrglue.py
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+import argparse
+import os
+import sys
+
+NAME_INDEX = 0
+HEADER_INDEX = 1
+COND_INDEX = 2
+RETTYPE_INDEX = 3
+PARM1_INDEX = 4
+
+POINTER_SIGNATURE = ['...', '*', '_sa_handler_t', ',mbstate_t']
+
+BLACK_LIST = ['sigqueue', 'pthread_create', 'pthread_detach',
+              'pthread_key_create', 'pthread_key_delete', 'pthread_getspecific',
+              'pthread_setspecific', 'mallinfo', 'inet_ntoa']
```

Comment:
i guess it's simpler to use whitelist as only very simple things can work.

Reply:
Maybe ture, there are many hand writing wrappers now, but we are trying to do it by tools.